activestorage/app/models/active_storage/blob.rb

```diff
@@ -313,7 +313,7 @@ def open(tmpdir: nil, &block)
   end
 
   def mirror_later # :nodoc:
-    service.mirror_later key, checksum: checksum if service.respond_to?(:mirror_later)
+    service.mirror_later(key) if service.respond_to?(:mirror_later)
```

Comment:
@skipkayhil You're calling `Service#mirror_later` without a required kwarg:
https://github.com/rails/rails/blob/189359f55a6c63e82bc3c2c57a35f9812c369841/activestorage/lib/active_storage/service/mirror_service.rb#L61

```
ActiveStorage::ManyAttachedTest#test_attaching_a_new_blob_from_an_uploaded_file_with_a_custom_service:
ArgumentError: missing keyword: :checksum
    lib/active_storage/service/mirror_service.rb:61:in `mirror_later'
    app/models/active_storage/blob.rb:316:in `mirror_later'
    app/models/active_storage/attachment.rb:131:in `mirror_blob_later'
```